components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/SAMLAssertionClaimsCallback.java

```diff
@@ -608,6 +609,10 @@ private void setClaimsToJwtClaimSet(Map<String, Object> claims, JWTClaimsSet jwt
         } finally {
             PrivilegedCarbonContext.endTenantFlow();
         }
+        if (resource != null && resource.getProperties() != null) {
```

Comment:
Shall we rename 'resource' to something like 'oidcScopesResource' so that its purpose is reflected easily?